pegasus/sites.v3/code.org/views/minecraft_2019.haml

```diff
@@ -76,8 +76,8 @@
   .clear
   %h2= I18n.t(:hoc2020_mc_available_mee)
```

Comment:
NIt: Could you update this file name to be `_2022`?

Reply:
Yep! I'll rename it to `minecraft_content` since it acts as a container; this way we don't have to change it every year.

Reply:
Good idea 👍